platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -119,6 +119,18 @@ MGL_EXPORT
  */
 @property (nonatomic) CGFloat scale;
 
+/**
+ :nodoc:
+ Whether the Mapbox wordmark is displayed.
+
+ @note The Mapbox terms of service, which governs the use of Mapbox-hosted
+ vector tiles and styles,
+ <a href="https://docs.mapbox.com/help/how-mapbox-works/attribution/">requires</a> most Mapbox
+ customers to display the Mapbox wordmark. If this applies to you, do not
+ hide the wordmark or change its contents.
```

Comment:
Ignoring any internal options for a moment and focusing on this property declaration alone, it wouldn’t be possible for the developer to hide the wordmark or change its contents, so why admonish against doing so?

Reply:
It is always possible for the developer to obscure the logo by blotting it out or clipping the snapshot. But I think a warning against doing that should be somewhere more prominent, like in the class documentation.